platform/backend/repository/sqlite/src/main/kotlin/io/hamal/repository/sqlite/ExecLog.kt

```diff
@@ -0,0 +1,162 @@
+package io.hamal.repository.sqlite
+
+import io.hamal.lib.sqlite.Connection
+import io.hamal.lib.sqlite.SqliteBaseRepository
+import io.hamal.repository.api.ExecLog
+import io.hamal.repository.api.ExecLogCmdRepository.AppendCmd
+import io.hamal.repository.api.ExecLogQueryRepository.ExecLogQuery
+import io.hamal.repository.api.ExecLogRepository
+import io.hamal.repository.sqlite.record.protobuf
+import kotlinx.serialization.ExperimentalSerializationApi
+import java.nio.file.Path
+
+@OptIn(ExperimentalSerializationApi::class)
+class ExecLogSqliteRepository(
+    config: Config
+) : SqliteBaseRepository(config), ExecLogRepository {
+    data class Config(
+        override val path: Path
+    ) : SqliteBaseRepository.Config {
+        override val filename = "exec_log.db"
+    }
+
+    override fun setupSchema(connection: Connection) {
+        connection.tx {
+            execute(
+                """
+                CREATE TABLE IF NOT EXISTS exec_log (
+                    idx         INTEGER PRIMARY KEY AUTOINCREMENT,
+                    id          INTEGER NOT NULL,
+                    exec_id     INTEGER NOT NULL,
+                    group_id    INTEGER NOT NULL,
+                    data        BLOB NOT NULL
```

Comment:
Gut feeling, ExecLog is a special case. Meaning, we dont store the blob - instead we store it entirely in the table, so for each field we have a column